linera-sdk/tests/fixtures/contract-call/res

```diff
@@ -0,0 +1,11 @@
+error: failed to load manifest for workspace member `/Users/mathieudutoursikiric/GITlineraOpen/GITlinera_sdk_tests/linera-protocol_second/linera-sdk-tests/create-and-call`
```

Comment:
That file was probably added by mistake?

Reply:
So, sorry.

Reply:
still there

Reply:
Thanks, forgotten to commit and push.